crates/vm/src/system/phantom/mod.rs

```diff
@@ -146,11 +146,11 @@ impl<F: PrimeField32> InstructionExecutor<F> for PhantomChip<F> {
                     pc: from_state.pc,
                     discriminant,
                 })?;
-            let mut streams = self.streams.get().unwrap().lock().unwrap();
+            let streams = self.streams.get().unwrap().lock().unwrap();
             sub_executor
                 .as_mut()
                 .phantom_execute(
-                    memory,
+                    unimplemented!(),
```

Comment:
This is not implemented because of trait object restrictions. We do not seem to use this though.